setuptools/command/easy_install.py

```diff
@@ -108,7 +108,7 @@ def isascii(s):
 else:
 
     def _to_ascii(s):
-        return s.encode('ascii')
+        return s.encode('utf8')
```

Comment:
Hm... Looking at what this does I think I agree with this change (though I don't know nearly enough about unicode issues to fully judge it), but maybe we should also change `_to_ascii` to `_to_bytes`?

Also, this change definitely needs tests.

Reply:
You mean having only the `_to_bytes()` function?

BTW, tests added

Reply:
Well just `_to_ascii` sounds like it turns something to an encoded ASCII string, but this is actually returning a utf-8-encoded byte string, so it should probably be called `_to_bytes()` instead of `_to_ascii()`.

Reply:
done